### PR TITLE
IGAPP-215: Changed the notFound key to "Page not found"

### PR DESCRIPTION
**Pull request by SimonVortkamp**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.
https://issues.tuerantuer.org/browse/IGAPP-215

### CODE REVIEW

**Review by klinzo — APPROVED** [review]
So you did check the translations of other languages as well, right?

**Comment by SimonVortkamp** [comment]
Now the translations should be right.

File: locales/locales.json [hunk 1]
```diff
@@ -216,7 +216,6 @@
         "disclaimer": "ናይ ደራሲ ዝርዝራት ሓበሬታን ናይ ርክብን",
         "mainDisclaimer": "ናይ ደራሲ ዝርዝራት ሓበሬታን ናይ ርክብን - {{appName}}",
         "pois": "ዘገድሱ ነጥብታት",
-        "notFound": "ገጽ ኣይተረኽበን",
```
Comment (maxammann) [inline locales/locales.json:219]: I think all translators used german as their source. This means the "Page not located" is an exception.

Did you use google translator here? If yes then I woudl say we leave the translation here.

**Review by maxammann — APPROVED** [review]

File: locales/locales.json (hunk 1 above)
Comment (SimonVortkamp) [reply locales/locales.json:219]: Yes, google translates this as "Sagen Sie uns nicht die Seite" or "Don't tell us the page", that's why I deleted it. I'm not sure how to check this translation.

Comment (maxammann) [reply locales/locales.json:219]: Me neighter :D but I'm oke with removing if we are not sure :+1:

**Pull request merged**